packages/jest-snapshot/src/snapshot_resolver.js

```diff
@@ -67,3 +71,21 @@ function mustImplement(functionName: string) {
     '\nDocumentation: https://facebook.github.io/jest/docs/en/configuration.html#snapshotResolver'
   );
 }
+
+function verifyConsistentTransformations(custom: SnapshotResolver) {
+  const fakeTestPath = path.join(
+    'some-path',
+    '__tests__',
+    'snapshot_resolver.test.js',
+  );
+  const transformedPath = custom.resolveTestPath(
+    custom.resolveSnapshotPath(fakeTestPath),
+  );
+  if (transformedPath !== fakeTestPath) {
```

Comment:
You can pass these 2 through `slash` to normalize the slashes

Reply:
I just did the thing I suggested in the edit, seems straight forward enough.

🤞 for CI